zebra-network/src/peer_set/initialize.rs

```diff
@@ -610,19 +612,37 @@ where
                 connection_tracker,
                 peerset_tx.clone(),
             )
-            .await?;
+            .await?
+            .map(move |res| match res {
```

Comment:
```suggestion
            .map(|res| match res {
```

Reply:
Thanks, I'll fix these in another PR, because I've actually seen this panic on mainnet, so I don't want our tests or unrelated PRs to start failing.